docs/contributer_guide/why_ngshare.rst

```diff
@@ -4,7 +4,49 @@ Why ngshare?
 The Problem
 -----------
 
-This section is under construction
+nbgrader can be used in JupyterHub for creating and grading assignments, but there are issues when JupyterHub is deployed as a Kubernetes cluster. nbgrader is unable to automatically distribute and collect assignments since there isn't a shared filesystem (used by the exchange directory). 
```

Comment:
```suggestion
nbgrader can be used in JupyterHub for creating and grading assignments, but there are issues when JupyterHub is deployed as a Kubernetes cluster. nbgrader distributes and collects assignments via a shared directory between instructors and students called the exchange directory. nbgrader does not work on a Kubernetes setup because there isn't a shared filesystem in which to place the exchange directory. 
```